lib/table_sync/publisher.rb

```diff
@@ -4,26 +4,27 @@ class TableSync::Publisher < TableSync::BasePublisher
   DEBOUNCE_TIME = 1.minute
 
   # 'original_attributes' are not published, they are used to resolve the routing key
-  def initialize(object_class, original_attributes, destroyed: nil, confirm: true, state: :updated)
```

Comment:
Publisher will now fail if state is not passed explicitly, let's use **opts, but set its values manually considering defaults